src/JustSaying/Extensions/AmazonSqsClientExtensions.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+
+namespace JustSaying.Extensions
+{
+    public static class AmazonSqsClientExtensions
+    {
+        public static async Task TagQueueAsync(this IAmazonSQS client, string queueUrl, Dictionary<string, string> tags, CancellationToken cancellationToken)
+        {
+            await client.TagQueueAsync(new TagQueueRequest()
```

Comment:
As public should null-check the client in both these methods. Does this extensions type need to be public though?

Reply:
No it probably doesn't need to be - will make this class internal as its just to make it easier to call these methods without having to know about or construct AWS request types.